Gemfile

```diff
@@ -30,7 +30,7 @@ end
 if puppetversion = ENV['PUPPET_GEM_VERSION']
   gem 'puppet', puppetversion, :require => false
 else
-  gem 'puppet', :require => false
+  gem 'puppet', '< 4.0.0', :require => false
```

Comment:
Actually, the odd thing is that line is `c.environmentpath = spec_path if Puppet.version.to_f >= 4.0`, so it's there specifically for puppet 4.